Detectors/ITSMFT/ITS/tracking/include/ITStracking/Tracker.h

```diff
@@ -61,41 +59,30 @@ class Tracker
   ~Tracker();
 
   void adoptTimeFrame(TimeFrame& tf);
-  void setBz(float bz);
-  float getBz() const;
 
   void clustersToTracks(
     std::function<void(std::string s)> = [](std::string s) { std::cout << s << std::endl; }, std::function<void(std::string s)> = [](std::string s) { std::cerr << s << std::endl; });
-  void clustersToTracksGPU(std::function<void(std::string s)> = [](std::string s) { std::cout << s << std::endl; });
-  void setSmoothing(bool v) { mApplySmoothing = v; }
-  bool getSmoothing() const { return mApplySmoothing; }
-
   std::vector<TrackITSExt>& getTracks();
 
-  void setCorrType(const o2::base::PropagatorImpl<float>::MatCorrType& type) { mCorrType = type; }
-  void setParameters(const std::vector<MemoryParameters>&, const std::vector<TrackingParameters>&);
+  void setParameters(const std::vector<TrackingParameters>&);
   void getGlobalConfiguration();
-  bool isMatLUT() const { return o2::base::Propagator::Instance()->getMatLUT() && (mCorrType == o2::base::PropagatorImpl<float>::MatCorrType::USEMatCorrLUT); }
-  // GPU-specific interfaces
-  TimeFrame* getTimeFrameGPU();
-  void loadToDevice();
+  void setBz(float);
+  void setCorrType(const o2::base::PropagatorImpl<float>::MatCorrType& type);
```

Comment:
this is just an enum, no point in passing by reference.

Reply:
Sure, changed.